vue-components/stories/Dialog.stories.ts

```diff
@@ -12,7 +12,7 @@ export function complex( args: { title: string, actions: string, dismissButton:
 		components: { Button, Dialog },
 		props: Object.keys( args ),
 		template: `
-			<div>
+			<div style="height: 70vh;">
```

Comment:
what does this fix?

Reply:
It increases the height of the canvas section in the Docs page, so the Dialog does not overlap with the props table (see my comment above).

Reply:
oh, yeah, just saw it now